File: ueueo-framework/ueueo-localization/src/main/java/Abp/Localization/LocalizationResourceExtensions.cs
using System;
using System.Collections.Generic;
using JetBrains.Annotations;
using Volo.Abp.Localization.VirtualFiles.Json;

namespace Volo.Abp.Localization;

public static class LocalizationResourceExtensions
{
    public static LocalizationResource AddVirtualJson(
        @NonNull this LocalizationResource localizationResource,
        @NonNull String virtualPath)
    {
        Objects.requireNonNull(localizationResource, nameof(localizationResource));
        Objects.requireNonNull(virtualPath, nameof(virtualPath));

        localizationResource.Contributors.Add(new JsonVirtualFileLocalizationResourceContributor(
            virtualPath.EnsureStartsWith('/')
        ));

        return localizationResource;
    }

    public static LocalizationResource AddBaseTypes(
        @NonNull this LocalizationResource localizationResource,
        @NonNull params Type[] types)
    {
        Objects.requireNonNull(localizationResource, nameof(localizationResource));
        Objects.requireNonNull(types, nameof(types));

        for (var type in types)
        {
            localizationResource.BaseResourceTypes.AddIfNotContains(type);
        }

        return localizationResource;
    }
}

File: ueueo-framework/ueueo-localization/src/main/java/com/ueueo/localization/LocalizationResourceExtensions.java
package com.ueueo.localization;

import com.ueueo.localization.virtualfiles.json.JsonVirtualFileLocalizationResourceContributor;
import org.springframework.lang.NonNull;

import java.util.Objects;

public class LocalizationResourceExtensions
{
    public static LocalizationResource addVirtualJson(
            @NonNull  LocalizationResource localizationResource,
            @NonNull String virtualPath)
    {
        Objects.requireNonNull(localizationResource);
        Objects.requireNonNull(virtualPath);

        localizationResource.getContributors().add(new JsonVirtualFileLocalizationResourceContributor(virtualPath));

        return localizationResource;
    }

    public static LocalizationResource addBaseTypes(
        @NonNull  LocalizationResource localizationResource,
        @NonNull  Class<?>... types)
    {
        Objects.requireNonNull(localizationResource);
        Objects.requireNonNull(types);

        for (Class<?> type : types)
        {
            localizationResource.getBaseResourceTypes().add(type);
        }
        return localizationResource;
    }
}
